### PR TITLE
chore: disable findTouchAliasElement logging

### PR DESCRIPTION
**Pull request by mcdurdin**
The `elementLogger()` function was causing a large number of events so I am
disabling this at least until we have time to investigate further.

### CODE REVIEW

File: web/source/dom/touchAliasElement.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@ namespace com.keyman.dom {
         console.info(`Target's grandparent: ${elementDescriber(target.parentElement.parentElement)}`);
       }
       // Triggers the actual Sentry log (on pages that include Sentry logging)
-      throw new Error(`Unexpected touch-start event target.`);
+      //throw new Error(`Unexpected touch-start event target.`);
```
Comment (jahorton) [inline web/source/dom/touchAliasElement.ts:80]: No `if(!keyman.isEmbedded)` check?  I'll admit I should have done that much originally; it'd still greatly help to get events from keymanweb.com in the meantime.

**Comment by darcywong00** [comment]
Noting this reverts part of #4821

File: web/source/dom/touchAliasElement.ts (hunk 1 above)
Comment (darcywong00) [reply web/source/dom/touchAliasElement.ts:80]: There's a few events from keymanweb.com that can be analyzed

http://sentry.keyman.com/organizations/keyman/issues/5444/?project=10&query=is%3Aunresolved

**Review by darcywong00 — APPROVED** [review]
lgtm

I'm OK with either disabling or adding the isEmbedded check

File: web/source/dom/touchAliasElement.ts (hunk 1 above)
Comment (mcdurdin) [reply web/source/dom/touchAliasElement.ts:80]: I think we should have some events from keymanweb.com. Given it's causing errors, let's not leave it running.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.36-alpha](https://keyman.com/downloads/releases/alpha/15.0.36)

**Pull request merged**